Sources/CoreCommands/Options.swift

```diff
@@ -470,6 +470,13 @@ public struct BuildOptions: ParsableArguments {
     )
     public var testEntryPointPath: AbsolutePath?
 
+    /// The lto mode to use if any.
+    @Option(
+        name: .customLong("experimental-lto-mode"),
+        help: .hidden
+    )
+    public var linkTimeOptimizationMode: LinkTimeOptimizationMode?
```

Comment:
Could this use that enum directly instead of creating a proxy enum, as they seem identical to each other?
```suggestion
    public var linkTimeOptimizationMode: BuildParameters.LinkTimeOptimizationMode?
```

Reply:
@neonichu suggested discrete enums, I'm happy to go either way.

Reply:
I think they should be separate because one is for argument parsing and one is for the build. They just happen to be the same right now, but that's coincidental not inherent.